### ISSUE TITLE
Get rid of m_staticText->Wrap( -1 );

### ISSUE THREAD

**Issue by Zenju** [wxFormBuilder/wxFormBuilder#357]
It seems the "m_staticText->Wrap( -1 );" calls that are automatically generated for all static text controls serve no purpose and just litter the code and waste (a few) CPU cycles.

**Comment by jhasse** [comment]
Are you sure?

**Comment by Zenju** [comment]
Yes, "wxStaticTextBase::Wrap(-1)" removes text wrapping if previously applied, but wxStaticTextCtrl, by default, is not wrapped.

**Comment by jhasse** [comment]
I see, thanks :)
